benchmarks/SourceTextBenchmarks.fs

```diff
@@ -0,0 +1,69 @@
+namespace Benchmarks
+
+open System
+open FSharp.Data.Adaptive
+open Microsoft.CodeAnalysis.Text
+type FileVersion = int
+
+
+module Helpers =
+  open FsAutoComplete.LspHelpers
+  open FSharp.UMX
+  open System.Collections.Generic
+
+  let fileContents = IO.File.ReadAllText(@"C:\Users\jimmy\Repositories\public\TheAngryByrd\span-playground\Romeo and Juliet by William Shakespeare.txt")
```

Comment:
Using this a just a giant text file. Should I include it or leave as an exercise for the others?